internal/attestation/aws/issuer_test.go

```diff
@@ -94,40 +103,17 @@ func TestGetInstanceInfo(t *testing.T) {
 	}
 }
 
-type fakeMetadataClient struct {
-	projectIDString    string
-	instanceNameString string
-	zoneString         string
-	projecIDErr        error
-	instanceNameErr    error
-	zoneErr            error
+type AWSMetadataAPIStub struct {
```

Comment:
```suggestion
type stubMetadataAPI struct {
```
Struct should not be exported.
For testing types we usually use the `stub<Type>` naming scheme